posthog/hogql_queries/query_runner.py

```diff
@@ -429,15 +432,26 @@ def apply_dashboard_filters(self, dashboard_filter: DashboardFilter) -> Q:
             query_update: dict[str, Any] = {}
             if dashboard_filter.properties:
                 if self.query.properties:
-                    query_update["properties"] = PropertyGroupFilter(
-                        type=FilterLogicalOperator.AND,
-                        values=[
-                            PropertyGroupFilterValue(type=FilterLogicalOperator.AND, values=self.query.properties),
-                            PropertyGroupFilterValue(
-                                type=FilterLogicalOperator.AND, values=dashboard_filter.properties
-                            ),
-                        ],
-                    )
+                    try:
+                        query_update["properties"] = PropertyGroupFilter(
+                            type=FilterLogicalOperator.AND,
+                            values=[
+                                PropertyGroupFilterValue(type=FilterLogicalOperator.AND, values=self.query.properties)
+                                if isinstance(self.query.properties, list)
+                                else PropertyGroupFilterValue(**self.query.properties.model_dump()),
+                                PropertyGroupFilterValue(
+                                    type=FilterLogicalOperator.AND, values=dashboard_filter.properties
+                                ),
+                            ],
+                        )
+                    except Exception as e:
+                        # If pydantic is unhappy about the shape of data, let's ignore property filters and carry on
+                        capture_exception()
```

Comment:
Curious: Do we have Sentry configured to capture exception-level log message as exceptions? If so, we don't need to capture separately, as below is sufficient.

Reply:
We don't, but there's a way: https://github.com/kiwicom/structlog-sentry